### PR TITLE
[DEST-869][CC-5311][WEBONISE][SEG-6] Add position to Google products

### PR DESCRIPTION
**Pull request by CarlosMecha**
Allow customers to provide the `position` field for products. Changes from #218

### CODE REVIEW

File: integrations/google-analytics/lib/index.js [hunk 1]
```diff
@@ -580,6 +580,25 @@ function formatValue(value) {
   return Math.round(value);
 }
 
+function getProductPosition(item, products) {
+  var position = item.properties().position;
+  if (
+    typeof position !== 'undefined' &&
+    !Number.isNaN(Number(position)) &&
+    Number(position) > -1
```
Comment (mericsson) [inline integrations/google-analytics/lib/index.js]: Should this also check `Number.isInteger` ?

File: integrations/google-analytics/lib/index.js [hunk 2]
```diff
@@ -580,6 +580,25 @@ function formatValue(value) {
   return Math.round(value);
 }
 
+function getProductPosition(item, products) {
+  var position = item.properties().position;
```
Comment (mericsson) [inline integrations/google-analytics/lib/index.js]: This seems safe. But is there ever a time that `position` is a property that is not intended to be used in this way?

Comment (CarlosMecha) [reply integrations/google-analytics/lib/index.js]: Not by our spec. This position is an attribute of a a Product.

File: integrations/google-analytics/lib/index.js (hunk 1 above)
Comment (CarlosMecha) [reply integrations/google-analytics/lib/index.js]: Good point, I'll make that extra check befire the release

Comment (CarlosMecha) [reply integrations/google-analytics/lib/index.js]: Sadly `Number.isInteger` is not compatible with early versions of Edge :S I'll leave to google to decide what to do with a floating point number.

**Pull request merged**